src/components/Button.tsx

```diff
@@ -107,14 +113,17 @@ const colorClasses = (type: ButtonType, available?: boolean) =>
       ? classnames(
           borderWidth('border'),
           borderRadius('rounded-full'),
-          borderColor(
-            'border-secondary',
-            'hover:border-secondary',
-            'active:border-secondary'
-          ),
+          borderColor({
+            'border-transparent': gradientFont,
+            'border-secondary': !gradientFont,
```

Comment:
I just think, that code goes through here and does conversion each time you do `!something`, maybe we should convert it in function body and pass as variable, wdyt?